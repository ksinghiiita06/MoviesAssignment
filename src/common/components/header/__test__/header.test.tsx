import configureStore from 'redux-mock-store';
import {render} from '@testing-library/react-native';
import React from 'react';
import Header from '..';
import {Provider} from 'react-redux';
import {SafeAreaProvider} from 'react-native-safe-area-context';

describe('Common Header', () => {
  const initialState = {
    appReducer: {languageCode: 'ar'},
  };
  const mockStore = configureStore();
  const store = mockStore(initialState);
  let wrapper: any;
  beforeEach(() => {
    wrapper = render(
      <SafeAreaProvider>
        <Provider store={store}>
          <Header />
        </Provider>
      </SafeAreaProvider>,
    );
  });
  it('Header renders correctly ', () => {
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
