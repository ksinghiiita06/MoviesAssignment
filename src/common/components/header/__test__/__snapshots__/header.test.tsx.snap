// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Common Header Header renders correctly  1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
